### ISSUE TITLE
speclib file format conversion

### ISSUE THREAD

**Issue by liangjin0912** [vdemichev/DiaNN#645]
DIA-NN generates in silico spectral library in .speclib format, is there a way to convert it to other formats like .tsv or .csv so it can be imported to other software? Thanks!

**Comment by vdemichev** [comment]
Yes, just check 'Generate spectral library' and keep the raw files field empty

**Comment by liangjin0912** [comment]
> Yes, just check 'Generate spectral library' and keep the raw files field empty

Perfect! Thank you so much for your quick response!
